README.md

```diff
@@ -61,3 +61,35 @@ Application should now be running on <a href="http://localhost:3006/order/">http
 ## Dependencies
 [Identity Server](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueIdentity).
 [Order Form API](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueOrdering#running-the-application).
+
+## Integration tests helpers
+Run the integration tests with `npm run test:integration` by default this will run the integration tests in 2 threads.
+
+The command args are now named which should made it easier to run particular tests
+
+### To run a particular browser `b`
+`npm run test:integration b=chrome` -> changes the browser to run as chrome. Default set to chrome:headless
+
+### To run ui tests for a particular page `p`
+`npm run test:integration p=catalogue=solutions/dashboard` -> Will run the 3 ui.test files within this page directory
+
+### To run a particular ui test file for a page `f`
+`npm run test:integration p=catalogue=solutions/dashboard f=general` -> Will run the general.ui.test within this page directory
+
+### Use the full path `fp`
+Right click on the test file and copy the path.
+`npm run test:integration fp=/Users/fawad.ali/projects/buying-catalogue/order-form/app/pages/sections/order-items/catalogue-solutions/dashboard/ui-tests/general.ui.test.js c=1`
```

Comment:
eek! apologies, probably not a good idea to leave your name in there

Reply:
good spot